treeherder/autoclassify/autoclassify.py

```diff
@@ -102,6 +103,12 @@ def find_best_matches(errors, matchers):
 
         yield best_match
 
+        newrelic.agent.record_custom_event('highest_scored_matcher', {
```

Comment:
I know it doesn't make any difference now (since `find_best_matches` is always fully consumed), but should this be before the yield in case in the future we do `first(find_best_matches)` or similar?

Reply:
Good idea, rebased!